src/test/consensus/Validations_test.cpp

```diff
@@ -480,6 +486,14 @@ class Validations_test : public beast::unit_test::suite
 
                 BEAST_EXPECT(harness.vals().numTrustedForLedger(ID{2}) == 0);
                 BEAST_EXPECT(harness.vals().numTrustedForLedger(ID{20}) == 1);
+
+                // A new key, but re-issue the same validation, since it is the
+                // same, it is considered a repeat
```

Comment:
I think we want this be `AddOutcome::current` for this case:
https://github.com/ripple/rippled/pull/2084#discussion_r113582740
We would need to change this check in `Validations::add`
https://github.com/ripple/rippled/pull/2084/files#diff-46f83c3cd15dd825b63068c0ab4dea00R383
to something like
```
auto const ret = byLedger_[id].emplace(key, val);
if (! ret.second && ret.first->second.key() == val.key())
    return AddOutcome::repeat;
```